packages/boxel-ui/addon/src/components/progress-bar/index.gts

```diff
@@ -118,6 +141,44 @@ export default class ProgressBar extends Component<Signature> {
           text-overflow: ellipsis;
           color: var(--progress-bar-font-color);
         }
+        .progress-circle {
+          --progress-bar-background-color: var(
+            --boxel-progress-bar-background-color,
+            var(--boxel-light-200)
+          );
+          --progress-bar-fill-color: var(
+            --boxel-progress-bar-fill-color,
+            var(--boxel-highlight)
+          );
+          --progress-bar-font-color: var(
+            --boxel-progress-bar-font-color,
+            var(--boxel-light)
+          );
+          --progress-bar-size: 80px;
```

Comment:
these are called progress-bar not progess-circle. For vars only related to circle then call it progress-circle. For other vars that may be shared, then you can move it to the outer-class I was talking about above